File: AD-HOC/2389 - Telescope/2389.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int A, N, F, views = 0;

        A = int.Parse(Console.ReadLine());
        N = int.Parse(Console.ReadLine());

        for (int i = 0; i < N; i++)
        {
            F = int.Parse(Console.ReadLine());
            if (F * A >= 40000000)
                views++;
        }
        Console.WriteLine(views);
    }
}

File: AD-HOC/2389 - Telescope/2389.c
#include <stdio.h>

int main()
{

    int A, N, F, views = 0;

    scanf("%i", &A);
    scanf("%i", &N);

    for (int i = 0; i < N; i++)
    {
        scanf("%i", &F);
        if (F * A >= 40000000)
            views++;
    }

    printf("%i\n", views);
    return 0;
}